CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,6 @@
+- Add User name, request path, query and body to AccessLogger (#541)
+- Ensure Origin of AccesLogger show x-forwarderd-for header if exists
+- Add Access Logger Patterns to match to Access Logger (#541)
+- Homogenizes format for Access Logger with pep logs
```

Comment:
```suggestion
- Add: user name, request path, query and body to AccessLogger (#541)
- Add: Access Logger Patterns to match to Access Logger (#541)
- Fix: ensure Origin of AccesLogger show x-forwarderd-for header if exists
- Fix: Homogenizes format for Access Logger with pep logs
```

Reply:
With regards to:

```
- Add: user name, request path, query and body to AccessLogger (#541)
```

maybe Service and SubService should be mentioned (in README.md fields with that names has been added to the bullet list...)?

With regards to "query", I don't see how it applies to the items added to the bullet list...

Reply:
> With regards to:
> 
> ```
> - Add: user name, request path, query and body to AccessLogger (#541)
> ```
> 
> maybe Service and SubService should be mentioned (in README.md fields with that names has been added to the bullet list...)?
> 
> With regards to "query", I don't see how it applies to the items added to the bullet list...

These are only new fields added. Service and SubService already are in AccessLogger